.rubocop.yml

```diff
@@ -35,6 +35,10 @@ Style/Documentation:
 Style/ModuleFunction:
   Enabled: false
 
+Style/NumericLiterals:
+  Exclude:
+    - 'spec/**/*'
```

Comment:
I'm not sure the best way to disable this just for Unix timestamps so I just disabled it entirely for specs